website/search/elastic_search.py

```diff
@@ -707,13 +707,13 @@ def update_user(user, index=None):
         'normalized_user': normalized_names['fullname'],
         'normalized_names': normalized_names,
         'names': names,
-        'job': user.jobs[0]['institution'] if user.jobs else '',
-        'job_title': user.jobs[0]['title'] if user.jobs else '',
-        'all_jobs': [job['institution'] for job in user.jobs[1:]],
-        'school': user.schools[0]['institution'] if user.schools else '',
-        'all_schools': [school['institution'] for school in user.schools],
+        #'job': user.jobs[0]['institution'] if user.jobs else '',
```

Comment:
Did you comment these out so that we have functioning code but to also allow them to serve as a placeholder for changes we will need to make to this file in the future?

Reply:
Yes, this is more of a TODO, as schools and jobs should be placed back in elastic search. I'll add a TODO note.